addons/mail/static/tests/mock_server/mock_models/mail_thread.js

```diff
@@ -65,16 +65,45 @@ export class MailThread extends models.ServerModel {
             ["res_model", "=", this._name],
             ["partner_id", "!=", this.env.user.partner_id],
         ];
-        if (after) {
-            domain.push(["id", ">", after]);
-        }
         if (filter_recipients) {
```

Comment:
@alexkuhn we've not implemented this `filter_recipients` filter at other places as well, if I do than I might need to change the test cases as well